BUILD

```diff
@@ -29,3 +29,35 @@ Maven repository for it. To build the binding:
         -Dpackaging=jar
 
 3. Uncomment `<module>nosqldb</module>` and run `mvn clean package`.
+
+# Mapkeeper 
+
+Mapkeeper binding doesn't get built by default because there is no
+Maven repository for it. To build the binding:
+
+1. follow "https://github.com/m1ch1/mapkeeper/wiki/Getting-Started" to Generate Thrift MapKeeper Binding
+   a.  install libevent
+   b.  install boost
+   c.  install thrift
+   d.  generate mapkeeper binding
+
+2. After generating mapkeeper binding, go the "mapkeeper/thrift/gen-java/target", and 
```

Comment:
There doesn't need to be a remote repo to install locally. Just use "mvn install" instead of "mvn package". Then other builds on the same machine will see the artifacts.

If you do set up a repo, please track your activity on #181.